core/arch/arm/kernel/abort.c

```diff
@@ -31,24 +31,10 @@ enum fault_type {
 static void get_current_ta_exidx_stack(vaddr_t *exidx, size_t *exidx_sz,
 				       vaddr_t *stack, size_t *stack_size)
 {
-	struct tee_ta_session *s;
-	struct user_ta_ctx *utc;
-
-	if (tee_ta_get_current_session(&s) != TEE_SUCCESS)
-		panic();
-
-	utc = to_user_ta_ctx(s->ctx);
-
-	/* Only 32-bit TAs use .ARM.exidx/.ARM.extab exception handling */
-	assert(utc->is_32bit);
-
-	*exidx = utc->exidx_start; /* NULL if TA has no unwind tables */
-	if (*exidx)
-		*exidx += utc->load_addr;
-	*exidx_sz = utc->exidx_size;
-
-	*stack = utc->stack_addr;
-	*stack_size = utc->mobj_stack->size;
+	*exidx = 0;
+	*exidx_sz = 0;
+	*stack = 0;
+	*stack_size = 0;
```

Comment:
The fact that this patch removes the call stack from TA abort dumps, to be re-introduced later, is worth mentioning in the commit description.

Reply:
I'll fix.